internal/utils/process/checksumfile.go

```diff
@@ -8,9 +8,13 @@ import (
 
 // Creates checksum file name according to template <image name:tag>.<algorithm>
 // e.g nginx:lates.md5
-func CheckSumFile(procName, alg string) string {
+func CheckSumFile(procName, alg string) (string, error) {
 	procImage := viper.GetStringMapString("process-image")
 	image := procImage[procName]
-
-	return fmt.Sprintf("%s.%s", image, strings.ToLower(alg))
+	ns := viper.GetString("pod-namespace")
+	parts := strings.Split(image, ":")
```

Comment:
You may use the `minio.BuildObjectName()` for consistency.

Reply:
I'm ok with this solution.